lms/djangoapps/mobile_api/users/serializers.py

```diff
@@ -140,38 +136,41 @@ def get_course_modes(self, obj):
             for mode in course_modes
         ]
 
-    def to_representation(self, instance):
+    def to_representation(self, instance: CourseEnrollment) -> 'OrderedDict':  # noqa: F821
         """
         Override the to_representation method to add the course_status field to the serialized data.
         """
         data = super().to_representation(instance)
-        if 'progress' in self.context.get('requested_fields', []):
-            data['progress'] = self.calculate_progress(instance)
+
+        if 'course_progress' in self.context.get('requested_fields', []) and self.context.get('api_version') == API_V4:
```

Comment:
Should we change/add "progress" to "course_progress" in a view that uses the `CourseEnrollmentSerializer` docstring?

Reply:
Yes, I forgot about that. I'm going to add this docstring